doc/jobs/ci_jobs.rst

```diff
@@ -124,69 +124,12 @@ Instead of invoking the generated script it can also be *sourced*:
 
 .. code:: sh
 
-  . ci_job_crystal.sh
+  . ci_job_rolling_ament_cmake_ros.sh
 
 The return code of the invocation of ``catkin_tests_results`` /
 ``colcon test-result`` is then available in the environment variable
 ``test_result_RC``.
 
-Run the *CI* job on Travis
```

Comment:
This section could be preserved / adapted to run via GitHub Actions. There are jobs in this very repo that use this.